frontend/mulighetsrommet-veileder-flate/src/components/sidemeny/SidemenyDetaljer.tsx

```diff
@@ -22,8 +22,8 @@ const SidemenyDetaljer = () => {
               Tiltaksnummer
             </BodyShort>
             <div className={styles.tiltaksnummer}>
-              <BodyShort size="small">{tiltaksnummer}</BodyShort>
-              <Kopiknapp kopitekst={String(tiltaksnummer)} dataTestId="knapp_kopier" />
+              <BodyShort size="small">{formaterTiltaksnummer(tiltaksnummer)}</BodyShort>
+              <Kopiknapp kopitekst={formaterTiltaksnummer(tiltaksnummer)} dataTestId="knapp_kopier" />
```

Comment:
Denne er litt interessant.

Jeg har en følelse av at det er fordelaktig å kun kopiere løpenummer-delen av tiltaksnummeret (basert på hvordan det er i Arena sitt GUI). Der er det også to input-felter og jeg tror ikke aar-feltet er påkrevd. Ved å kun kopiere andre halvdel slipper man steg 3 i følgende arbeidsflyt:
1.  copy tiltaksnummer
2. paste tiltaksnummer i løpenummer-felt iarena
3. fjern år og skilletegn
4. søk

Kanskje greit å få bekreftet?

Reply:
Koden over vil bare kopiere løpenummeret fra tiltaksnummer. Kanskje litt misvisende navngivning på `formaterTiltaksnummer`-metoden ser jeg.

Reply:
Ah, litt forvirrende ja, kanskje greit å tydeligjøre at at vi utleder løpenummer i stedet :smile: